packages/react-components/src/theming/v9ThemeShim.ts

```diff
@@ -0,0 +1,230 @@
+    colorNeutralStrokeOnBrand2: palette.white,
+    colorNeutralStrokeOnBrand2Hover: palette.white,
+    colorNeutralStrokeOnBrand2Pressed: palette.white,
+    colorNeutralStrokeOnBrand2Selected: palette.white,
+    colorBrandStroke1: palette.themePrimary,
+    colorBrandStroke2: palette.themeLight,
+    colorBrandStroke2Hover: palette.themeLight,
+    colorBrandStroke2Pressed: palette.themeLight,
+    colorBrandStroke2Contrast: palette.themeLight,
+    colorCompoundBrandStroke: palette.themePrimary,
+    colorCompoundBrandStrokeHover: palette.themeDarkAlt,
+    colorCompoundBrandStrokePressed: palette.themeDark,
+    colorNeutralStrokeDisabled: palette.neutralQuaternaryAlt,
+    colorNeutralStrokeInvertedDisabled: whiteAlpha[40],
+    colorTransparentStroke: 'transparent',
+    colorTransparentStrokeInteractive: 'transparent',
+    colorTransparentStrokeDisabled: 'transparent',
+    colorNeutralStrokeAlpha: inverted ? whiteAlpha[10] : blackAlpha[5],
+    colorNeutralStrokeAlpha2: whiteAlpha[20],
+    colorStrokeFocus1: palette.white,
+    colorStrokeFocus2: palette.black,
+    colorNeutralShadowAmbient: 'rgba(0,0,0,0.12)',
+    colorNeutralShadowKey: 'rgba(0,0,0,0.14)',
+    colorNeutralShadowAmbientLighter: 'rgba(0,0,0,0.06)',
+    colorNeutralShadowKeyLighter: 'rgba(0,0,0,0.07)',
+    colorNeutralShadowAmbientDarker: 'rgba(0,0,0,0.20)',
+    colorNeutralShadowKeyDarker: 'rgba(0,0,0,0.24)',
+    colorBrandShadowAmbient: 'rgba(0,0,0,0.30)',
+    colorBrandShadowKey: 'rgba(0,0,0,0.25)'
+  };
+};
+
+/**
+ * Creates v9 shadow tokens from v8 effects.
+ */
+const mapShadowTokens = (effects: IEffects): Partial<ShadowTokens> => {
+  return {
+    shadow4: effects.elevation4,
+    shadow8: effects.elevation8,
+    shadow16: effects.elevation16,
+    shadow64: effects.elevation64
+  };
+};
+
+/**
+ * Creates v9 border radius tokens from v8 effects
+ */
+const mapBorderRadiusTokens = (effects: IEffects): Partial<BorderRadiusTokens> => {
+  return {
+    borderRadiusSmall: effects.roundedCorner2,
+    borderRadiusMedium: effects.roundedCorner4,
+    borderRadiusLarge: effects.roundedCorner6
+  };
+};
+
+/**
+ * Creates a v9 theme from a v8 theme and base v9 theme.
+ * FluentUI webLightTheme is used in case if no baseThemeV9 is provided.
+ *
+ * @private
+ */
+export const createV9Theme = (themeV8: ThemeV8, baseThemeV9?: ThemeV9): ThemeV9 & { errorText: string } => {
+  const baseTheme = baseThemeV9 ?? webLightTheme;
+  return {
+    ...baseTheme,
+    ...mapAliasColors(themeV8.palette, themeV8.isInverted),
+    ...mapShadowTokens(themeV8.effects),
+    ...mapBorderRadiusTokens(themeV8.effects),
+    colorBrandBackground2: themeV8.palette.themeLight, // updated from palette.themeLighterAlt
+    colorBrandBackground2Hover: themeV8.palette.themeLight, // updated from palette.themeLighterAlt
+    colorBrandBackground2Pressed: themeV8.palette.themeLight, // updated from palette.themeLighterAlt
+    colorStatusWarningBackground3: '#D83B01',
```

Comment:
Is this color always static even in dark vs light vs high contrast?

Reply:
Yes, it is the one that was used for mentions

Reply:
is there a concern if contoso wants to override this they can't because of the hard coding?

Reply:
not now as we haven't made it public before. So, when we decide to change this, we can reuse v8 color here.